packages/node_modules/@webex/plugin-meetings/src/constants.js

```diff
@@ -66,6 +70,9 @@ export const TYPE = 'type';
 
 export const USE_URI_LOOKUP_FALSE = 'useUriLookup=false';
 
+export const VALID_PMR_ADDRESS = /([a-z0-9][-a-z0-9, '.']{0,62})@([a-z0-9][-a-z0-9, '.']{0,62})\.webex\.com/i;
```

Comment:
Do you have a regex reference for these like in your other PR? I liked that you included that.

Reply:
I don't for all of them, some of them just coming from web client ;/